### PR TITLE
Ensure accordion uses overriden focus colour

### PR DESCRIPTION
**Pull request by NickColley**
At the moment if you override the focus colour and focus this component this icon will remain yellow.
By using the `$govuk-focus-colour` variable we allow this to change as expected.

Before:

Focused accordion with yellow inside the chevron icon

After:

Focused accordion with overriden colour inside the chevron icon

### CODE REVIEW

File: CHANGELOG.md
Comment (nickcolley) [inline CHANGELOG.md:85]: me thanking myself 😂

**Review by 36degrees — APPROVED** [review]
This change makes sense to me – thanks @nickcolley 👍🏻 

Just needs a second review from the team and then we can get this merged.

**Comment by owenatgov** [comment]
This PR is now unexpectedly closed, despite it being a helpful an inoffensive change 🤔 Github hasn't recorded who closed it which isn't super helpful. I'm going to be bold and reopen. @NickColley Did you close this? If you did, did you mean to or was it an accident.

**Review by owenatgov — APPROVED** [review]
🪗

**Pull request merged**
